staging/src/k8s.io/client-go/rest/request.go

```diff
@@ -129,7 +129,7 @@ func NewRequest(c *RESTClient) *Request {
 		backoff = c.createBackoffMgr()
 	}
 	if backoff == nil {
-		backoff = noBackoff
+		backoff = defaultBackoffManager()
```

Comment:
If we want to start using exponential backoff, I would like this TODO to be addressed:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/rest/with_retry.go#L225-L229

[ideally in a separate PR first, but might be separate commit here]

I would like to avoid increasing every sleep by additional 1s...

Reply:
yes, i would like to address this in a follow up PR - we already have an issue tracking this.

Reply:
So my point is - we need to address that one first - otherwise this PR automatically means that in the first retry we wait 2s instead of 1s.

We have to merge the PR fixing that issue not later than this PR...

Reply:
https://github.com/kubernetes/kubernetes/pull/110100